components/_common_/Section/Section.js

```diff
@@ -47,9 +47,8 @@ function Section({
     // Dark background colors have light heading color
     case 'gray':
     case 'slate':
-      headingTheme = 'white';
-      break;
     default:
+      headingTheme = 'white';
```

Comment:
I'd prefer this change get reverted so that we're explicit in the casings. If another theme gets added this could cause an obfuscated headache. Currently a new theme will break the component which is ideal so a dev is forced to explicitly handle the case.

Reply:
ok, no problem